### ISSUE TITLE
Fontactive callbacks returns true even if it should false (sometimes)

### ISSUE THREAD

**Issue by pixedelic** [typekit/webfontloader#235]
Hi, I'm building a small UI to load Google fonts on a WP theme. Some select boxes allow to select the font family, the option selected enables the ability to select among the available font weight but for font style I would like to use the "fontinactive" callbacks... but, sometime it doesn't seem to work.

I made a simpler example here https://jsfiddle.net/pixedelic/zL5two28/5/

To replicate the issue select:

- "Offside"
- "400"
- "italic"

The alert returns the fontinactive message since the italic style doesn't exist.

Then select:

- "Offside"
- "400"
- "normal"

The alert returns all the active messages since the regular style exists.

Then select again:

- "Offside"
- "400"
- "italic"

It shouldn't return the "fontinactive" message, but it returns all the active ones... am I missing something? Thank you in advance for any advice! Manuel :-)

**Comment by LL782** [comment]
Sounds like an interesting UI in the making. I know I'd make use of that.

I found something relevant in the readme:

> The wf-inactive class is applied only if none of the fonts on the page have rendered. Otherwise, the wf-active class is applied (even if some fonts are inactive).

So I guess that's covers this issue. It seems a bit disappointing that API won't give a response for the specific call being made. I suppose it's a scenario that hadn't been considered before.

**Comment by pixedelic** [comment]
Yes, indeed, a bit disappointing. I can't control that in any way, but it should be easy (I guess, but I'm far from being an expert of webfontloader) to add a callback that returns whether a particular font with its variants is available or not. The browser itself returns the error, even if I don't know how to use this, since if I tried, for instance, if a particular font is available via AJAX, I'd get the usual error " XMLHttpRequest cannot load... etc. etc.".

Maybe there is an easier way I don't see at this moment... I could check the presence of a particular style starting from the font family and the weight selected, using the API, but it will increase the amount of functions considerably...
